src/ray/raylet/lib/java/org_ray_runtime_raylet_RayletClientImpl.cc

```diff
@@ -9,21 +9,14 @@
 template <typename ID>
 class UniqueIdFromJByteArray {
  public:
-  const ID &GetId() const { return *id_pointer_; }
+  inline const ID &GetId() const { return id; }
```

Comment:
We needn't explicitly declare `inline` for a member method which defined in a class.

Reply:
ok. But I think either way is fine.